tests/unit/commands/init/test_cli.py

```diff
@@ -627,14 +627,14 @@ def test_init_cli_int_with_image_app_template(
 
         # 1: AWS Quick Start Templates
         # 2: Package type - Image
-        # 12: Java8 base image
+        # 13: Java8 base image
         # 1: dependency manager maven
         # test-project: response to name
 
         user_input = """
 1
 2
-12
+13
```

Comment:
Can we also check if we have same tests in integration tests which will require input to generate the project?

Reply:
I didn't find any other changes here and other places.